gauntlet/packages/gauntlet-solana/src/commands/internal/solana.ts

```diff
@@ -63,6 +68,38 @@ export default abstract class SolanaCommand extends WriteCommand<TransactionResp
     }
   }
 
+  signAndSendRawTx = async (rawTxs: RawTransaction[], extraSigners?: Keypair[]): Promise<TransactionSignature> => {
+    const latestSlot = await this.provider.connection.getSlot()
+    const recentBlock = await this.provider.connection.getBlock(latestSlot)
+    const tx = makeTx(rawTxs, {
+      recentBlockhash: recentBlock.blockhash,
+      feePayer: this.wallet.publicKey,
+    })
+    if (extraSigners) {
+      tx.sign(...extraSigners)
+    }
+    const signedTx = await this.wallet.signTransaction(tx)
+    logger.loading('Sending tx...')
+    return await sendAndConfirmRawTransaction(this.provider.connection, signedTx.serialize())
+  }
+
+  withIDL = (action: (...args: any) => Promise<TransactionSignature>, idl: Idl) => async (
+    ...args
+  ): Promise<TransactionSignature> => {
+    try {
+      return await action(...args)
+    } catch (e) {
+      console.error(e)
```

Comment:
Will this printout cause duplicated error logs, as this function logs and throws again to be handled.

Consider removing this line.